cleanlab/classification.py

```diff
@@ -292,7 +293,7 @@ def fit(
           The classifier that this instance was initialized with,
           ``clf``, must be able to fit() and predict() data with this format.
 
-        labels : np.array or pd.Series
+        y : np.array or pd.Series
```

Comment:
Note there are many other instances in the docstrings throughout this file (particularly near the top) which reference variable \`labels\`. We need to update all these references to be \`y\` now.

Reply:
Note I just changed all these from labels to y for the 2.0 release and now that 2.0 is released you are recommending changing them all back. Let's discuss.

Reply:
@jwmueller Thanks for pointing out the module docstring. References have been fixed there.
The only other references are scoped to different functions like `find_label_issues` and `_process_label_issues_arg`.
Shouldn't I leave probabilities like 
https://github.com/cleanlab/cleanlab/blob/5973a21958f47fd8e7246ccf36c8304ff4f22e3b/cleanlab/classification.py#L301
,
https://github.com/cleanlab/cleanlab/blob/5973a21958f47fd8e7246ccf36c8304ff4f22e3b/cleanlab/classification.py#L692
as is?


---


@cgnorthcutt Hopefully we can "keep" `labels` as a type hint later on.
See https://github.com/cleanlab/cleanlab/pull/317#issuecomment-1193201089.